### PR TITLE
Working vagrant VM for OpenMBEE MMS

### PR DESCRIPTION
**Pull request by manuel-j-diaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,18 +7,22 @@ A vagrant-based virtual machine (VM) for setting up the [Open Model Based Engine
...
 
 2. Install [VirtualBox][virtualbox].
 
-3. Clone this repository:
+
+## Installation
+1. Clone this repository:
     ```
-    git clone https://github.com/sanbales/openmbee-vm.git
+    git clone https://github.com/MJDiaz89/openmbee-vm.git
```
Comment (sanbales) [inline README.md]: Manny, para el PR, podes cambiar esto a la original?

**Review by sanbales — CHANGES_REQUESTED** [review]
This looks great.  Only think I'd ask is the README if we merge it back to my repo, to change the URL.

Many thanks for making all these fixes!

**Comment by manuel-j-diaz** [comment]
Si, claro. Good catch! I'll fix soon.

File: README.md (hunk 1 above)
Comment (manuel-j-diaz) [reply README.md]: OK, my latest commit reverts this line back to your URL

**Pull request merged**